dgbpy/hdf5.py

```diff
@@ -249,33 +249,36 @@ def getLogDir( withtensorboard, examplenm, platform, basedir, clearlogs, args ):
   logdir = logdir / Path(jobnm+str(nrsavedruns+1)+'_'+'m'.join( datetime.now().isoformat().split(':')[:-1] ))
   return logdir
 
-def getOutdType( classinfo ):
-  max = classinfo.max()
-  min = classinfo.min()
-  min_abs = np.abs(min)
-  max_abs = np.abs(max)
-  if max_abs>min_abs:
-    pass
+def getOutdType( classinfo, withunlabaled='No' ):
+  if withunlabaled == 'Yes':
+    return np.int16
   else:
-    max_abs = min_abs
-  if min < 0 and max > 0:
-    if max_abs < 129:
-      return np.int8
-    elif max_abs > 128 and max_abs < 32769:
-      return np.int16
-    elif max_abs > 32768 and max_abs < int(2.2e10):
-      return np.int32
+    max = classinfo.max()
```

Comment:
The else statement is not required, because if the preceeding statement is true, you will have returned from the function, so anything below is not executed